File: DiagramasUML_Proy2/DiagramasUML_Proy2/Jugador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiagramasUML_Proy2
{
    class Jugador:Persona
    {
        private int id;
        private string conference;
        private string team;
        private string position;
        private int power;
        private string skill;
        private int height;
        private int scoredPoints;
        private int stolenBalls;
        private int rebounds;
        private int plugs;
        private int votes;       

        public int getId()
        {
            return id;
        }
        public void setId(int id)
        {
            this.id = id;
        }
        public string getConference()
        {
            return conference;
        }
        public void setConference(string conference)
        {
            this.conference = conference;
        }
        public string getTeam()
        {
            return team;
        }
        public void setTeam(string team)
        {
            this.team = team;
        }
        public string getPosition()
        {
            return position;
        }
        public void setPosition(string position)
        {
            this.position = position;
        }
        public int getPower()
        {
            return power;
        }
        public void setPower(int power)
        {
            this.power = power;
        }
        public string getSkill()
        {
            return skill;
        }
        public void setSkill(string skill)
        {
            this.skill = skill;
        }
        public int getHeight()
        {
            return height;
        }
        public void setHeight(int height)
        {
            this.height = height;
        }
        public int getScoredPoints()
        {
            return scoredPoints;
        }
        public void setScoredPoints(int scoredPoints)
        {
            this.scoredPoints = scoredPoints;
        }
        public int getStolenBalls()
        {
            return stolenBalls;
        }
        public void setStolenBalls(int stolenBalls)
        {
            this.stolenBalls = stolenBalls;
        }
        public int getRebounds()
        {
            return rebounds;
        }
        public void setRebounds(int rebounds)
        {
            this.rebounds = rebounds;
        }
        public int getPlugs()
        {
            return plugs;
        }
        public void setPlugs(int plugs)
        {
            this.plugs = plugs;
        }
        public int getVotes()
        {
            return votes;
        }
        public void setVotes(int votes)
        {
            this.votes = votes;
        }

        public string toString()
        {
            return "Jugador [id=" + id + ", conference=" + conference + ", team=" + team + ", position=" + position
                    + ", power=" + power + ", skill=" + skill + ", height=" + height + ", scoredPoints=" + scoredPoints
                    + ", stolenBalls=" + stolenBalls + ", rebounds=" + rebounds + ", plugs=" + plugs + ", votes=" + votes
                    + "]";
        }
    }
}

File: Proyecto2_1096917/Proyecto2_1096917/Proyecto2_1096917/Jugador.cpp
#include "pch.h"
#include "Jugador.h"

Jugador::Jugador() {}

Jugador::Jugador(string conference, string division, string city, string team, string position, int number, int id,
	string name, string height, string weight, string birthdate, int power, string skill, int scoredPoints,
	int stolenBalls, int rebounds, int plugs, int votes) {
	this->setName(name);	
	this->id = id;
	this->birthdate = birthdate;
	this->city = city;
	this->division = division;
	this->conference = conference;
	this->team = team;
	this->position = position;
	this->power = power;
	this->skill = skill;
	this->height = height;
	this->scoredPoints = scoredPoints;
	this->stolenBalls = stolenBalls;
	this->rebounds = rebounds;
	this->plugs = plugs;
	this->votes = votes;
}

int Jugador::getId()
{
	return id;
}

void Jugador::setId(int jId)
{
	id = jId;
}

int Jugador::getNumber() {
	return number;
}

void Jugador::setNumber(int jNumber) {
	number = jNumber;
}

string Jugador::getConference()
{
	return conference;
}

void Jugador::setConference(string jConference)
{
	conference = jConference;
}

string Jugador::getTeam()
{
	return team;
}

void Jugador::setTeam(string jTeam)
{
	team = jTeam;
}

string Jugador::getPosition()
{
	return position;
}

void Jugador::setPosition(string jPosition)
{
	position = jPosition;
}

string Jugador::getWeight(){
	return weight;
}

void Jugador::setWeight(string jWeight) {
	weight = jWeight;
}

string Jugador::getBirthdate() {
	return birthdate;
}

void Jugador::setBirthDate(string jBirthdate) {
	birthdate = jBirthdate;
}

int Jugador::getPower()
{
	return power;
}

void Jugador::setPower(int jPower)
{
	power = jPower;
}

string Jugador::getSkill()
{
	return skill;
}

void Jugador::setSkill(string jSkill)
{
	skill = jSkill;
}

string Jugador::getHeight()
{
	return height;
}

void Jugador::setHeight(string jHeight)
{
	height = jHeight;
}

int Jugador::getScoredPoints()
{
	return scoredPoints;
}

void Jugador::setScoredPoints(int jScoredPoints)
{
	scoredPoints = jScoredPoints;
}

int Jugador::getStolenBalls()
{
	return stolenBalls;
}

void Jugador::setStolenBalls(int jStolenBalls)
{
	stolenBalls = jStolenBalls;
}

int Jugador::getRebounds()
{
	return rebounds;
}

void Jugador::setRebounds(int jRebounds)
{
	rebounds = jRebounds;
}

int Jugador::getPlugs()
{
	return plugs;
}

void Jugador::setPlugs(int jPlugs)
{
	plugs = jPlugs;
}

int Jugador::getVotes()
{
	return votes;
}

void Jugador::setVotes(int jVotes)
{
	votes = jVotes;
}

void Jugador::toString() {	
	cout << "Jugador [ID: " << id << ", Nombre: " << getName() << ", Ciudad: " << city << ", Conferencia: " << conference << ", Equipo: " << team << ", Posicion:" << position
		<< ", Potencia: " << power << ", Habilidad: " << skill << ", Altura: " << height << ", Puntos: " << scoredPoints
		<< ", Balones robados=" << stolenBalls << ", Rebotes: " << rebounds << ", Tapones: " << plugs << ", Votos: " << votes
		<< ", Peso: " << weight << ", Numero: " << number << ", Fecha de Nacimiento: " << birthdate << "]";
}

Jugador::~Jugador()
{
}
